d1985r0.md

```diff
@@ -19,6 +19,15 @@ type traits.
 
 # Change Log
 
+## R1 -> R2
+
+Having found overwhelming support for the feature in EWGI, and a concern
+about contra- and co-variance of the `template auto` parameters, we include
+discussion of these topics in the paper, together with the decision and
+comparison tables that underpin the decision.
+
+The decision was to leave the co/contravariance as is.
+
 ## R0 -> R1
 
 - Greatly expanded the number of examples based on feedback from the BSI panel.
```

Comment:
Hm, you don't seem to be able to comment outside the diff, but here you go.

Directly under motivation I find the writing a bit hard to understand. "a metafunction for apply" could be easier to read if formulated "a metafunction like apply". I think you can remove the next sentence, it just causes confusion.

pure types could be rephrased as a template taking only types as parameters.

Reply:
Thanks! Reformulated.